src/MoBi.Presentation/DTO/ModuleConfigurationDTO.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using MoBi.Assets;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+
+namespace MoBi.Presentation.DTO
+{
+   public class ModuleConfigurationDTO
+   {
+      private readonly ModuleConfiguration _moduleConfiguration;
+      private readonly List<MoleculeStartValuesBuildingBlock> _moleculeStartValuesCollection = new List<MoleculeStartValuesBuildingBlock> { NullMoleculeStartValues };
+      private readonly List<ParameterStartValuesBuildingBlock> _parameterStartValuesCollection = new List<ParameterStartValuesBuildingBlock> { NullParameterStartValues };
+
+      public ModuleConfigurationDTO(ModuleConfiguration moduleConfiguration)
+      {
+         _moduleConfiguration = moduleConfiguration;
+         SelectedMoleculeStartValues = moduleConfiguration.SelectedMoleculeStartValues ?? NullMoleculeStartValues;
+         SelectedParameterStartValues = moduleConfiguration.SelectedParameterStartValues ?? NullParameterStartValues;
+         _moleculeStartValuesCollection.AddRange(moduleConfiguration.Module.MoleculeStartValuesCollection);
+         _parameterStartValuesCollection.AddRange(moduleConfiguration.Module.ParameterStartValuesCollection);
+      }
+
+      public ParameterStartValuesBuildingBlock SelectedParameterStartValues { get; set; }
+      public MoleculeStartValuesBuildingBlock SelectedMoleculeStartValues { get; set; }
+      public IReadOnlyList<MoleculeStartValuesBuildingBlock> MoleculeStartValuesCollection => _moleculeStartValuesCollection;
+      public IReadOnlyList<ParameterStartValuesBuildingBlock> ParameterStartValuesCollection => _parameterStartValuesCollection;
+
+      public static MoleculeStartValuesBuildingBlock NullMoleculeStartValues { get; } = new MoleculeStartValuesBuildingBlock().WithName(AppConstants.Captions.NoMoleculeStartValues);
```

Comment:
I find it very hard to reason about to have static members between various class members. Can you move them higher up and separate them clearly. Also you are accessing them before they are declared. This can create issues sometimes